### DISCUSSION TITLE
Can we add steps  on how to get the AI to read the files.

### DISCUSSION THREAD

**Discussion by Fire3473** [samuelbutler/wonda#1] [General]
I'd really like to have a guide here explaining the process on how to implement these files and get AI to use them

**Comment by samuelbutler** [comment]
Can you maybe give some more examples of what you're looking for, that is generalize-able to anyone using this template. A lot of the process for how to implement these files and get the AI to use them, other than what is already in the README, is usually use-case specific. 

Let me know if there is anything in specific you would like me to add to the README.

**Reply by nathangur** [reply]
I have two things I want it to read. I want it to read a api docs.json file and some code that is 2 folders down. I want it to code based on these scripts, but it gets very VERY confused and repeats itself over and over.

**Reply by samuelbutler** [reply]
is the code that is "2 folders down" outside of the auto_gpt_workspace folder? if so, the agent won't be able to access those files. come to the #prompt-engineering channel on the Discord and I can help you with your specific issue 
https://discord.gg/autogpt

**Comment by nathangur** [comment]
No this is all in the auto_gpt_workspace folder. I'll be there!
